parser_email.py

```diff
@@ -794,7 +794,23 @@ def _add_body_in_email_headers(parsed_mail, file_path, charset, content_type):
             body_content = f.read()
         _debug_print("Reading file data using binary mode")
     # Add body to the last added Email artifact
-    body_content = UnicodeDammit(body_content).unicode_markup.encode('utf-8').decode('utf-8')
+    body_content = UnicodeDammit(body_content).unicode_markup.encode('utf-8').decode('utf-8').replace('\u0000', '')
+
+    _debug_print('Processing email part with content_type: %s' % content_type)
+
+    IMAGE_CONTENT_TYPES = ['image/jpeg', 'image/png']
+
+    if any(t for t in IMAGE_CONTENT_TYPES if t in content_type):
```

Comment:
didn't track all the places `content_type` can come from, so can `content_type` be `None`? cause if so then doing `... in None` would raise an `Exception`

Reply:
Good point. I've added check to account for the None content_type.